### PR TITLE
allow additional openms peak picking

### PR DESCRIPTION
**Pull request by jpfeuffer**
Since sometimes TRFP misses to pick

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -406,6 +408,35 @@ process generate_decoy_database {
...
+     file "*.log"
+
+    script:
+     // TODO maybe allow specifying ms-levels and inmemory
+     """
+     PeakPickerHiRes -in ${mzml_file} \\
+                     -out ${mzml_file.baseName}_picked.mzML \\
+                     -threads ${task.cpus} \\
+                     -debug ${params.pp_debug} \\
+                     -processOption lowmemory \\
```
Comment (timosachsenberg) [inline main.nf:434]: does the low memory also detect if it is picked?
the in memory has the "auto" mode

Comment (jpfeuffer) [reply main.nf:434]: it uses PeakPickerHiRes.pick()
https://github.com/OpenMS/OpenMS/blob/develop/src/topp/PeakPickerHiRes.cpp#L149

Comment (jpfeuffer) [reply main.nf:434]: no it is not:
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp#L525
Why is this if-case not inside the pick function?

Comment (timosachsenberg) [reply main.nf:434]: hmm maybe because of the templated code?
Should still be possible to put it in the pick for MSSpectrum

Comment (jpfeuffer) [reply main.nf]: ok but anyway. nothing we can do here except allowing inmemory. which I did.

**Pull request merged**